app/controllers/books_controller.rb

```diff
@@ -0,0 +1,9 @@
+class BooksController < ApplicationController
+  include Wor::Paginate
+  before_action :authenticate_user!
+  def show; end
+
+  def index
+    render_paginated Book
```

Comment:
Book is model name. If you like to render all books, you should use 
```rb
Book.all
```

Reply:
Done